File: question/lintcode/lintcode0175/lintcode0175.js
function InvertBinaryTree(root) {
  if (root === null) {
    return;
  }
  let tmp = root.Left;
  root.Left = root.Right;
  root.Right = tmp;
  InvertBinaryTree(root.Left);
  InvertBinaryTree(root.Right);
}

File: question/lintcode/lintcode0175/lintcode0175.go
package lintcode

import "clarencedu/algorithm-go/structure"

type TreeNode = structure.TreeNode

func InvertBinaryTree(root *TreeNode) {
	if root == nil {
		return
	}
	root.Left, root.Right = root.Right, root.Left
	InvertBinaryTree(root.Left)
	InvertBinaryTree(root.Right)
}